### DISCUSSION TITLE
PlanToRun not recognized

### DISCUSSION THREAD

**Discussion by arronuk28** [soranoo/TradingView-Free-Webhook-Alerts#44] [Q&A] [answered]
I keep coming up with an error, PlanToRun not recognized. Iv been trying for hours to fix it with the assistance of Chatgpt aswell. but without success. Can you please tell me how to get around this, I have tried on both windows and Linux,  I have went from python3 to python 3.11 its driving me crazy. its to webhook Tradingview to IQ options

**Comment by soranoo** [comment, marked as answer]
@arronuk28 Thank you for your report. The issues have been fixed in the latest release.

Reference Issue #45
